dashboards/flutter-android/flutter-android.json

```diff
@@ -0,0 +1,1077 @@
+					"visualization": {
+						"id": "viz.area"
+					},
+					"rawConfiguration": {
+						"facet": {
+							"showOtherSeries": false
+						},
+						"legend": {
+							"enabled": true
+						},
+						"nrqlQueries": [
+							{
+								"accountIds": [],
+								"query": "SELECT percentage(uniqueCount(uuid), where exceptionLocation IS NOT NULL) as 'Users affected' FROM MobileSession, MobileHandledException LIMIT 1000 TIMESERIES WHERE appName = {{appName}}"
+							}
+						],
+						"platformOptions": {
+							"ignoreTimeRange": false
+						}
+					}
+				},
+				{
+					"title": "Dart Sessions Affected Rate Due to Exception Location",
+					"layout": {
+						"column": 1,
+						"row": 5,
+						"width": 6,
+						"height": 3
+					},
+					"linkedEntityGuids": null,
+					"visualization": {
+						"id": "viz.stacked-bar"
+					},
+					"rawConfiguration": {
+						"facet": {
+							"showOtherSeries": false
+						},
+						"legend": {
+							"enabled": true
+						},
+						"nrqlQueries": [
+							{
+								"accountIds": [],
+								"query": "SELECT percentage(uniqueCount(sessionId), where exceptionLocation IS NOT NULL) as 'Sessions affected' FROM MobileSession, MobileHandledException LIMIT 1000 TIMESERIES WHERE appName = {{appName}}"
+							}
+						],
+						"platformOptions": {
+							"ignoreTimeRange": false
+						}
+					}
+				},
+				{
+					"title": "Recent Dart Exception Causes",
+					"layout": {
+						"column": 7,
+						"row": 5,
+						"width": 6,
+						"height": 3
+					},
+					"linkedEntityGuids": null,
+					"visualization": {
+						"id": "viz.bar"
+					},
+					"rawConfiguration": {
+						"facet": {
+							"showOtherSeries": false
+						},
+						"nrqlQueries": [
+							{
+								"accountIds": [],
+								"query": "SELECT count(*) FROM MobileHandledException FACET `exceptionCause` LIMIT 5 WHERE appName = {{appName}}"
+							}
+						],
+						"platformOptions": {
+							"ignoreTimeRange": false
+						}
+					}
+				}
+			]
+		},
+		{
+			"name": "Alerts",
+			"description": null,
+			"widgets": [
+				{
+					"title": "Crash Percent (%)",
+					"layout": {
+						"column": 1,
+						"row": 1,
+						"width": 3,
+						"height": 3
+					},
+					"linkedEntityGuids": null,
+					"visualization": {
+						"id": "viz.billboard"
+					},
+					"rawConfiguration": {
+						"facet": {
+							"showOtherSeries": false
+						},
+						"nrqlQueries": [
+							{
+								"accountIds": [],
+								"query": "SELECT percentage(uniqueCount(sessionId), where category LIKE 'Crash') as '' FROM MobileSession, MobileCrash LIMIT 5 WHERE appName = {{appName}}"
+							}
+						],
+						"platformOptions": {
+							"ignoreTimeRange": false
+						},
+						"thresholds": [
+							{
+								"alertSeverity": "CRITICAL",
+								"value": 0.05
+							},
+							{
+								"alertSeverity": "WARNING",
+								"value": 0.02
+							}
+						]
+					}
+				},
+				{
+					"title": "Error & Failure Percent (%)",
```

Comment:
```suggestion
					"title": "Error & Failure Rate (%)",
```

Reply:
I have updated it as per your comments.
Thank you...